<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UniViewToolbarUI</class>
 <widget class="QWidget" name="UniViewToolbarUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="main_la">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QToolButton" name="rotate">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../share/icons/share_icons.qrc">
       <normaloff>:/share_icons/blue/image-rotate.svg</normaloff>:/share_icons/blue/image-rotate.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="fit_mode">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../share/icons/share_icons.qrc">
       <normaloff>:/share_icons/blue/shrink.svg</normaloff>:/share_icons/blue/shrink.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="zoom_out">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../share/icons/share_icons.qrc">
       <normaloff>:/share_icons/blue/zoom-out.svg</normaloff>:/share_icons/blue/zoom-out.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="zoom_in">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../share/icons/share_icons.qrc">
       <normaloff>:/share_icons/blue/zoom-in.svg</normaloff>:/share_icons/blue/zoom-in.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="scale">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::PlusMinus</enum>
     </property>
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="page_spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="page_lb">
     <property name="text">
      <string>Страница</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="page_la">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QSpinBox" name="page_no_ed">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="page_no_lb">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="page_div">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="page_max">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="right_spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../share/icons/share_icons.qrc"/>
 </resources>
 <connections/>
</ui>
